src/main/resources/db/changelog/alert/5.2.0/changelog-configuration-timestamp.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="jrichard" id="2019-10-07-15-15-58-375-1">
+         <addColumn tableName="DESCRIPTOR_CONFIGS" schemaName="ALERT">
+             <column name="CREATED_AT" type="TIMESTAMP" valueComputed="SYSDATE"/>
+        </addColumn>
+        <addColumn tableName="DESCRIPTOR_CONFIGS" schemaName="ALERT">
+            <column name="LAST_UPDATED" type="TIMESTAMP" valueComputed="SYSDATE"/>
+        </addColumn>
+    </changeSet>
```

Comment:
Should we consider adding Timestamp data to all configurations in the upgrade? This way all pages will have the created/modified data present when viewing the UI instead of random pages showing this data and others not.

Reply:
The defaultValueComputed="SYSDATE" attribute on the column may handle creating a default value of the current time.